src/modules/items/client.ts

```diff
@@ -97,6 +97,10 @@ export class ItemsClient {
       }
     }
 
+    if (filters.isWearableSmart) {
```

Comment:
Would you mind adding a test in the client for this filter?

Reply:
Added!